client/network/src/protocol/sync.rs

```diff
@@ -306,13 +327,65 @@ pub enum OnBlockData<B: BlockT> {
 	Request(PeerId, BlockRequest<B>)
 }
 
-/// Result of [`ChainSync::on_block_announce`].
+/// Result of [`ChainSync::process_block_announce_pre_validation`].
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub enum OnBlockAnnounce {
+pub enum BlockAnnounceResult<H> {
```

Comment:
```suggestion
pub enum BlockAnnounce<H> {
```

None of the other results below (e.g. `PreValidateBlockAnnounce`) are postfixed with `Result`. Would it not make sense to stay consistent?

Reply:
Done